### DISCUSSION TITLE
How to get computation cost and communication cost in distributed training?

### DISCUSSION THREAD

**Discussion by ZHAIXINGZHAIYUE** [apache/incubator-mxnet#15734] [Q&A]
How to get computation cost and communication cost in distributed training? Then, i could to improve training effiency.

**Comment by vdantu** [comment]
@mxnet-label-bot  add [question, training]
@apeforest @yuxihu : Would you be able to help answer this question?

**Comment by apeforest** [comment]
@ZHAIXINGZHAIYUE Please run profiler following: https://mxnet.incubator.apache.org/versions/master/tutorials/python/profiler.html
